elasticsearch/setup_es.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need both `setup.sh` and `setup_es.sh`?

Reply:
:+1: